.eslintrc.js

```diff
@@ -10,11 +10,9 @@ const sharedPlugins = [
 const sharedExtends = [
   'standard',
   'plugin:react/recommended',
-  'plugin:prettier/recommended',
   'plugin:jest-playwright/recommended',
   'prettier',
-  'prettier/react',
-  'prettier/standard',
```

Comment:
never found a reference to `prettier/standard` and `prettier/react` is now part of `prettier`.